### ISSUE TITLE
Cannot read property 'content' of undefined

### ISSUE THREAD

**Issue by JohnatanNorman** [cferdinandi/gumshoe#105]
Not sure what happens, but I was using gatsbyjs and your other library (smooth-scroll) and it works wonders. Both of them were required the same way: I was checking if window object (gatsbyjs thing) exists and registering gumshoe like so
```
const Gumshoe = require('gumshoejs');
const spy = new Gumshoe('#sidenav a');
```
sidenav is of course the id of ul. Any ideas what is causing that?

UPDATE: var doesn't change anything of course, because it's scoped to the page.

**Comment by cferdinandi** [comment]
Reduced test case please: https://github.com/cferdinandi/gumshoe/blob/master/.github/contributing.md

**Comment by cferdinandi** [comment]
Closing due to non-response. Will reopen if a reduced test case is provided.

**Comment by mikemartin** [comment]
@JohnatanNorman I had this error when my `#sidenav` ID didn't exist it my template, causing the `navItems` array to be empty.

**Comment by matthew-ia** [comment]
Also having this issue. 

**Edit:** I initially supplied test case in this comment, but I've found it was invalid, so I've removed it. Will open a new issue if I can accurately reproduce. I do believe this may be related to frameworks which mount/remount, e.g. React (OP mentioned Gatsby). I'm using Svelte, and the issue I'm experiencing actually occurs when a link inside of the spy list has an `href` that is an id, but that id doesn't exist anywhere on the page.
